packages/terra-markdown/package.json

```diff
@@ -27,7 +27,7 @@
   "dependencies": {
     "classnames": "^2.2.5",
     "github-markdown-css": "^2.4.1",
-    "prismjs": "~1.15.0",
+    "prismjs": "~1.16.0",
```

Comment:
I'm keeping the ~ for as long as we keep refractor as a dependency to avoid duplicate dependencies.

Reply:
What other package pulls this in? Doc-template via `react-syntax-highlighter`?

Reply:
Yes exactly